### PR TITLE
Added MarkDown to HTML assignment

### PR DESCRIPTION
**Pull request by Moibrahi7**

### CODE REVIEW

File: lesson_01/MohamedIbrahim/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel =  "Stylesheet" href="stylesheet.css">
+    <title>Mohamed Ibrahim Read Me</title>
+</head>
+<body>
+        <div class = "intro">
+            <h>👋Hi, I'm Mohamed</h>
```
Comment (anthonydmays) [inline lesson_01/MohamedIbrahim/index.html:11]: There's some differences between your HTML and your MARKDOWN version, please fix.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Review by anthonydmays — CHANGES_REQUESTED** [review]
I think you're moving too quickly. You still have significant differences between your two versions (some sections seem to be missing, along with some bullet points).

**Comment by Moibrahi7** [comment]
I have made changes to both the html readme and the Markdown readme respectfully to make them match and also word them better -MI

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request merged**